neurom/analysis/morphtree.py

```diff
@@ -237,11 +237,11 @@ def get_bounding_box(tree):
     [xmax, ymax, zmax]]
     """
 
-    min_x = min(p[0] for p in val_iter(tr.ipreorder(tree)))
-    min_y = min(p[1] for p in val_iter(tr.ipreorder(tree)))
-    min_z = min(p[2] for p in val_iter(tr.ipreorder(tree)))
-    max_x = max(p[0] for p in val_iter(tr.ipreorder(tree)))
-    max_y = max(p[1] for p in val_iter(tr.ipreorder(tree)))
-    max_z = max(p[2] for p in val_iter(tr.ipreorder(tree)))
+    min_x = min(p[COLS.X] for p in val_iter(tr.ipreorder(tree)))
```

Comment:
At the cost of building a numpy array. I'll try profiling it. The thing is, I don't know what kind of data sizes to expect. I also intend to memoize most Neuron methods, because it is supposed to be read-only.